nf1_cellpainting_env.yml

```diff
@@ -7,6 +7,7 @@ dependencies:
 # this is restricted to Python 3.8 at this time due to other conflicts
 - conda-forge::python=3.8
 - conda-forge::ipykernel
+- conda-forge::jupyterlab
```

Comment:
Can you remind me what jupyterlab is for here?

Also, are you still having issues with getting this environment downloaded?

Reply:
> Can you remind me what jupyterlab is for here?

I use jupyter lab for development :)

> Also, are you still having issues with getting this environment downloaded?

Yes, I think the issue is the same one Dayna was having with centrosome. Do you remember the fix?

Reply:
I think the issues were resolved in a recent release of CellProfiler 4.2.6 in which centrosome was causing issues for all 4.0 versions (IIRC). This should be solved if you try now. Let me know if not.